software/test_project/src/game/Main.java

```diff
@@ -17,11 +18,14 @@ public Main() {
 		InputManager inputManager = new InputManager();
 		Map map = new Map();
 		GraphicsManager gm = new GraphicsManager(inputManager, map); //create the panel
+
+		MapSaver.createMap();
```

Comment:
What is this supposed to do?
Why should it create a map when the game starts? Isn't this already done in the Map.java??